src/js/server/schemaLinkHandler.js

```diff
@@ -0,0 +1,47 @@
+/*
+
+    An extension of `gpii.express.schemaLink.handler` that sends the appropriate JSON Schema headers on success or failure. See
+    the component's documentation for more details:
+
+    https://github.com/the-t-in-rtf/gpii-json-schema/blob/master/docs/schemaLinks.md
+
+ */
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("gpii-express");
+require("../common/hasRequiredOptions");
+require("./schemaUrlHolder");
+require("./lib/schemaLinkHeaders");
+
+fluid.registerNamespace("gpii.schema.schemaLink.handler");
+
+gpii.schema.schemaLink.handler.sendResponse = function (that, response, statusCode, body) {
+    gpii.schema.schemaLinks.addHeaders(response, that.options.schemaUrls.success);
```

Comment:
This is an absolutely shining candidate for being implemented by an event-driven workflow with namespaced listeners, rather than working by "override and copy" - see above comments

Reply:
I think I agree, but it seems more in scope for a major (or at least minor) release of gpii-express, as it would require updating everything that uses the current gpii-handler grade.  @amb26, would it be enough for now to write up a ticket for discussion and leave a comment here?

Reply:
I hope so, because I wrote one up: [GPII-2632](https://issues.gpii.net/browse/GPII-2632).

Reply:
Yup, that is great